atom/browser/native_window_views_win.cc

```diff
@@ -165,6 +165,9 @@ bool NativeWindowViews::PreHandleMSG(
       }
       return false;
     }
+    case WM_WINDOWPOSCHANGING: {
+      return !CanResize();
```

Comment:
I think we should instead do something like this:

```cpp
if (!CanResize()) {
  const auto* window_pos = reinterpret_cast<WINDOWPOS*>(l_param);
  window_pos->flags |= SWP_NOSIZE;
}
```

Otherwise we might unintentionally ignore e.g. x/y/z position changes.

Reply:
I've tested this and found that (at least on Windows10) `SWP_NOSIZE` blocks the 'BrowserWindow.setSize()' calls, but still lets the window resize on taskbar resize, which is the exact opposite of what I want to achieve. Moving the window seems to be working by hand and from code with the original method too, which is strange, since I'm basically saying that the event is handled. I'm open for suggestions for better solutions for this.

Reply:
Hmm, interesting! Could you check if the size is retained with something like:

```cpp
case WM_GETMINMAXINFO: {
  const auto* min_max_info = reinterpret_cast<MINMAXINFO*>(l_param);
  min_max_info->ptMinTrackSize.x = some_width;
  min_max_info->ptMinTrackSize.y = some_height;
  return false;
}
```

If that works, maybe we could store the proper size and use it there (and update the stored size when `BrowserWindow.setSize()` is called).

Reply:
That seems to be working, but only if I return true in the end (stating that we handled the event). Do you think that could interfere with something?